client/src/components/Category/style.css

```diff
@@ -1,10 +1,17 @@
+* {
+    padding: 0;
+    margin: 0;
+}
```

Comment:
@OnsNader 
I think you don't need these, since these styles are already in app.css:
https://github.com/FACG5/CHariZone/blob/8f9485d7d4b2ad7e5cd29a4e8cd54f1a8d9eae83/client/src/components/App.css#L3

.. I was just about to suggest that you move these into app.css or a more "global" location, since they apply to all elements, but then I found them there already 😀